packages/clerk-js/src/ui/primitives/Button.tsx

```diff
@@ -159,7 +159,7 @@ const { applyVariants, filterProps } = createVariants((theme, props: OwnProps) =
           '&:active': { backgroundColor: theme.colors.$transparent },
         },
         link: {
-          ...common.textVariants(theme).buttonSmall,
+          // ...common.textVariants(theme).buttonSmall,
```

Comment:
Why we commented out this one?

Reply:
This will be removed entirely from the `link` variant, as we can't also apply a text variant afterwards.

Example with code randomly picked from the codebase:
```
<Button
   variant='link'
   textVariant='buttonLarge'
   onClick={() => setDisplayFormat('uri')}
   localizationKey={localizationKeys(
     'userProfile.mfaTOTPPage.authenticatorApp.buttonUnableToScan__nonPrimary',
   )}
/>
```

The example above will ignore `textVariant='buttonLarge'` and will keep the `textVariant` as `buttonSmall`.